certified-connectors/tyntec-WA/apiDefinition.swagger.json

```diff
@@ -2,13 +2,13 @@
   "swagger": "2.0",
   "info": {
     "title": "tyntec-wa",
```

Comment:
@ridlees Can you please update it to `tyntec wa` or `tyntec WA` as per the suggestions I have provided in the portal.